src/Compilers/Core/MSBuildTask/InteractiveCompiler.cs

```diff
@@ -194,17 +216,15 @@ protected override int ExecuteTool(string pathToTool, string responseFileCommand
 
         protected override string GenerateCommandLineCommands()
         {
-            var commandLineBuilder = new CommandLineBuilderExtension();
-            AddCommandLineCommands(commandLineBuilder);
-            return commandLineBuilder.ToString();
+            return DotnetHostInfo.CommandLineArgs;
```

Comment:
This means `Csc.CommandLineArgs` will now include the `csc.dll` argument when running on CoreClr correct? I think that makes sense but I want to make sure @agocke considers that explicitly.